components/automate-ui/package.json

```diff
@@ -65,42 +68,45 @@
     "zone.js": "^0.10.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.1002.0",
-    "@angular/cli": "^10.2.0",
-    "@angular/compiler-cli": "^10.2.3",
-    "@angular/language-service": "^10.2.3",
-    "@types/faker": "^4.1.12",
-    "@types/jasmine": "=3.3.10",
+    "@angular-devkit/build-angular": "^0.1100.5",
+    "@angular/cli": "^11.0.5",
+    "@angular/compiler-cli": "^11.0.5",
+    "@angular/language-service": "^11.0.5",
+    "@types/faker": "^5.1.5",
+    "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "^2.0.8",
     "@types/lodash": "^4.14.165",
-    "ajv": "^6.12.6",
+    "ajv": "^7.0.2",
     "angular2-template-loader": "^0.6.2",
     "axe-webdriverjs": "^2.3.0",
-    "codelyzer": "^6.0.1",
-    "faker": "^4.1.0",
+    "codelyzer": "^6.0.0",
+    "faker": "^5.1.0",
     "hock": "^1.4.1",
-    "htmlhint": "^0.11.0",
+    "htmlhint": "^0.14.2",
     "install": "^0.13.0",
-    "jasmine-core": "~3.5.0",
-    "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~5.0.9",
+    "jasmine-core": "~3.6.0",
+    "jasmine-spec-reporter": "~6.0.0",
+    "karma": "~5.1.1",
     "karma-chrome-launcher": "~3.1.0",
     "karma-cli": "^2.0.0",
     "karma-coverage-istanbul-reporter": "~3.0.2",
-    "karma-firefox-launcher": "^1.3.0",
+    "karma-firefox-launcher": "^2.1.0",
     "karma-jasmine": "~4.0.0",
-    "karma-jasmine-html-reporter": "^1.5.4",
+    "karma-jasmine-html-reporter": "^1.5.0",
     "karma-safari-launcher": "^1.0.0",
     "ng2-mock-component": "^0.2.0",
-    "node-sass": "^4.14.1",
     "protractor": "~7.0.0",
     "rxjs-spy": "^7.5.3",
     "rxjs-tslint": "^0.1.8",
+    "sass": "^1.30.0",
     "sass-lint": "^1.13.1",
-    "ts-node": "^8.10.2",
-    "tslint": "^5.0.0",
+    "ts-node": "^9.1.1",
+    "tslint": "^6.1.3",
     "tslint-defocus": "^2.0.6",
     "typescript": "~4.0.5",
     "wallaby-webpack": "^3.9.16"
+  },
+  "resolutions": {
+    "minimist": "^1.2.5"
   }
```

Comment:
This forces any packages depending on `minimist` to use `^1.2.5` - which help solve our `minimist` security vulnerability